jmix-flowui/flowui/src/main/java/io/jmix/flowui/bulk/BulkEditorDataProvider.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2023 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.flowui.bulk;
+
+import io.jmix.core.DataManager;
+import io.jmix.core.FetchPlan;
+import io.jmix.core.LoadContext;
+import io.jmix.core.entity.EntityValues;
+import io.jmix.core.metamodel.model.MetaClass;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component("flowui_BulkEditorDataProvider")
+public class BulkEditorDataProvider {
```

Comment:
`DataProvider` is more for UI component and their `setItems`/`setDataProvider` methods. Let's name it `BulkEditorDataLoadSupport` similar to DataLoad delegate.